.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crl
```

Comment:
Is the last line supposed to end with `eol=crl`?

I don't really use Windows at all, but my understanding is the two possible attributes for `eol` are `crlf` or `lf`.

See https://www.git-scm.com/docs/gitattributes#_eol

Reply:
typo!!! thanks!